packages/foundation-models/README.md

```diff
@@ -17,9 +17,54 @@ $ npm install @sap-ai-sdk/foundation-models
   - Create a `.env` file in the sample-code directory.
   - Add an entry `AICORE_SERVICE_KEY='<content-of-service-key>'`.
 
-## Usage
+## OpenAI client
 
-<!-- Details to be added -->
+The OpenAI client can be used to send chat completion or embedding requests to the OpenAI model deployed in SAP Generative AI Hub.
+
+### Prerequisites
+
+- A deployed OpenAI model in SAP Generative AI Hub.
+  - You can use the [`DeploymentApi`](../ai-api/README.md#deploymentapi) from `@sap-ai-sdk/ai-api` to deploy a model to SAP Generative AI Hub.
+- `sap-ai-sdk/foundation-models` package installed in your project.
+
+### Chat completion Client Usage
+
+```TS
```

Comment:
[req] There is no description for this example. Could you add that? I am also missing a general note on deployment resolution (you could also link there if needed).